OpenRA.Game/Support/Log.cs

```diff
@@ -75,22 +77,33 @@ public static void AddChannel(string channelName, string baseFilename)
 			}
 		}
 
-		public static void Write(string channel, string value)
+		public static void Write(string channelName, string value)
 		{
-			var writer = Channel(channel).Writer;
+			var channel = Channels[channelName];
+			var writer = channel.Writer;
 			if (writer == null)
 				return;
 
-			writer.WriteLine(value);
+			if (!channel.IsTimestamped)
+				writer.WriteLine(value);
+			else
+				writer.WriteLine("[{0}] {1}", DateTime.Now.ToString(Game.Settings.Server.TimestampFormat), value);
```

Comment:
Please write this as
```csharp
else
{
	var timestamp = DateTime.Now.ToString(Game.Settings.Server.TimestampFormat);
	writer.WriteLine("[{0}] {1}", timestamp, value);
}
```

to improve readability.

Reply:
Done.